src/lib.rs

```diff
@@ -49,10 +49,10 @@ pub mod dsp_source;
 /// # use bevy::prelude::*;
 /// # use bevy_fundsp::prelude::*;
 /// App::new()
-///     .add_plugins(DefaultPlugins)
-///     .add_plugin(DspPlugin::default())
+///     .add_plugins((DefaultPlugins, DspPlugin::default()))
 ///     .run()
 /// ```
+#[allow(dead_code)]
```

Comment:
Why exactly is this lint here? Just asking.

Reply:
I think `sample_rate` only gets set via the tests.

without this Rust reports, 

```
warning: field `sample_rate` is never read
  --> src/lib.rs:56:5
   |
55 | pub struct DspPlugin {
   |            --------- field in this struct
56 |     sample_rate: f32,
   |     ^^^^^^^^^^^
   |
   = note: `#[warn(dead_code)]` on by default
```

Let me know if there is another or better way to resolve this warning.